systemuser.sh

```diff
@@ -331,12 +331,7 @@ fi
 # including loading any .pth files which modify sys.path
 
 log_info "Running the notebook server"
+# Force the old backend, since the newer version uses jupyter-server by default
+export JUPYTERHUB_SINGLEUSER_APP='notebook'
```

Comment:
Separate commit please.

Reply:
I'll do it, but I do not fully agree. The reason we need this is because we update the base image, so they could be tied together.